junit-platform-console/src/main/java/org/junit/platform/console/ConsoleLauncher.java

```diff
@@ -30,9 +30,7 @@
 public class ConsoleLauncher {
 
 	public static void main(String... args) {
-		PrintWriter out = new PrintWriter(System.out);
-		PrintWriter err = new PrintWriter(System.err);
```

Comment:
Why not "just" use `new PrintWriter(System.out|.err, true);` here?

Reply:
Because theoretically, we'd also have to determine the correct encoding (which Picocli already does using the `sun.stdout.encoding` and `sun.stderr.encoding` system properties).